pkg/config/templates/rancherd-13-monitoring.yaml

```diff
@@ -31,8 +31,12 @@ bootstrapResources:
         prometheusSpec:
           evaluationInterval: 1m
           resources:
+            limits:
+              cpu: 1000m
+              memory: 3000Mi
             requests:
-              cpu: 500m
+              cpu: 750m
+              memory: 750Mi
```

Comment:
per [Rancher's monitoring doc](https://rancher.com/docs/rancher/v2.6/en/monitoring-alerting/configuration/helm-chart-options/#configuring-resource-limits-and-requests) I think we may need to update those values accordingly.
<img width="1078" alt="image" src="https://user-images.githubusercontent.com/4569037/157363022-b5699c53-4e65-4f77-a642-8ad961b7168f.png">

Reply:
updated in https://github.com/harvester/harvester-installer/commit/299c22aefde185ce98cade0d46499e86fe66a4b2